packages/block-library/src/cover/edit/inspector-controls.js

```diff
@@ -160,72 +195,139 @@ export default function CoverInspectorControls( {
 		),
 	};
 
+	const dropdownMenuProps = useToolsPanelDropdownMenuProps();
+
 	return (
 		<>
 			<InspectorControls>
 				{ !! url && (
-					<PanelBody title={ __( 'Settings' ) }>
+					<ToolsPanel
+						label={ __( 'Settings' ) }
+						resetAll={ () => {
+							setAttributes( {
+								hasParallax: undefined,
```

Comment:
```suggestion
								hasParallax: false,
```
If we do not explicitly specify a default value, a warning error will occur:

![image](https://github.com/user-attachments/assets/b7954380-e620-4693-bcfa-9cb1f59ae884)